app-react/src/main/kotlin/com/ustadmobile/mui/components/UstadAddListItem.kt

```diff
@@ -0,0 +1,52 @@
+package com.ustadmobile.mui.components
+
+import mui.icons.material.Add
+import mui.material.*
+import react.FC
+import react.Props
+import react.ReactNode
+import react.create
+
+external interface UstadAddListItemProps: Props {
+
+    var text: String
+
+    var enabled: Boolean?
+
+    var icon: ReactNode
```

Comment:
We want the add icon to be the default, but we can't set default values on an external interface. 

You can achieve the same effect by making this nullable:

e.g. var icon: ReactNode?

then in ListItemIcon use:

+ (props.icon ?: Add.create() )